installers/server/release/server.sh

```diff
@@ -113,9 +114,11 @@ if [ ! -z $SERVER_LISTEN_HOST ]; then
     GO_SERVER_SYSTEM_PROPERTIES="$GO_SERVER_SYSTEM_PROPERTIES -Dcruise.listen.host=$SERVER_LISTEN_HOST"
 fi
 SERVER_STARTUP_ARGS+=("-server $YOURKIT")
+SERVER_STARTUP_ARGS+=("-XX:+IgnoreUnrecognizedVMOptions")
 SERVER_STARTUP_ARGS+=("-Xms$SERVER_MEM -Xmx$SERVER_MAX_MEM -XX:PermSize=$SERVER_MIN_PERM_GEN -XX:MaxPermSize=$SERVER_MAX_PERM_GEN")
+SERVER_STARTUP_ARGS+=("-XX:MaxMetaspaceSize=$SERVER_MAX_METASPACE")
 SERVER_STARTUP_ARGS+=("$JVM_DEBUG $GC_LOG $GO_SERVER_SYSTEM_PROPERTIES")
-SERVER_STARTUP_ARGS+=("-Duser.language=en -Djruby.rack.request.size.threshold.bytes=30000000")
+SERVER_STARTUP_ARGS+=("-Duser.language=en -Djruby.rack.request.size.threshold.bytes=30000000 -Djruby.compile.invokedynamics=false")
```

Comment:
Thought so but went by this comment.
https://github.com/jruby/jruby/issues/1858#issuecomment-50159599

It is ```invokedynamic``